projects/web-codegen/src/lib/web-aggregator.service.spec.ts

```diff
@@ -1,63 +1,114 @@
 import { TestBed } from '@angular/core/testing';
 import { WebAggregatorService } from './web-aggregator.service';
+import {
+  TextService,
+  SymbolService,
+  ImageService,
+  LayerService,
+  FormatService
+} from '@xlayers/sketch-lib';
+import { CssCodeGenService } from '@xlayers/css-codegen';
+import { WebContextService } from './web-context.service';
+import { SvgCodeGenService } from '@xlayers/svg-codegen';
+import { WebCodeGenContext } from '@xlayers/web-codegen/lib/web-codegen';
 describe('WebAggregatorService', () => {
-    let service: WebAggregatorService;
+  let webAggregatorService: WebAggregatorService;
+  const webContext: WebCodeGenContext = {
+    attributes: [],
+    components: []
+  };
 
-    beforeEach(() => {
-        TestBed.configureTestingModule({
-            providers: [WebAggregatorService]
-        });
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        WebAggregatorService,
 
-        service = TestBed.get(WebAggregatorService);
-    });
+        // @TODO find a better way to mock this service
+        TextService,
+        // {
+        //   provide: TextService,
+        //   useValue: { identify: () => false, lookup: () => '' }
+        // },
```

Comment:
Maybe we should create a issue for this?

Reply:
responded.